src/azul/service/responseobjects/hca_response_v5.py

```diff
@@ -222,9 +224,15 @@ def _push_content_single_part(self) -> str:
 
         :return: S3 object key
         """
+        if self.format == 'tsv':
+            data = self._construct_tsv_content().encode()
+            content_type = 'text/tab-separated-values'
+        else:
```

Comment:
Every possible value of `self.format` should be explicitly tested for and the else branch should read:

```
else:
    assert False
```

Reply:
fixed